scale/scheduler/sync/job_type_manager.py

```diff
@@ -80,10 +80,12 @@ def sync_with_database(self):
         updated_job_types = {}
         for job_type in JobType.objects.all().iterator():
             try:
+                job_type.title = job_type.get_title()
+                job_type.description = job_type.get_description()
                 updated_job_types[job_type.id] = job_type
                 update_job_type_resources.append(job_type.get_resources())
             except InvalidSeedMetadataDefinition as ex:
-                logger.info('Invalid Seed manifest for job type %s-%s, id=%d' % (job_type.name, job_type.version, job_type.id))
+                logger.info('Invalid Seed manifest for job type %s-%s, id=%d\n Exception: %s' % (job_type.name, job_type.version, job_type.id, ex))
```

Comment:
I'd recommend changing this to a `logger.exception` from `logger.info`. That eliminates the need to manually inject the exception and will provide the entire stack trace rather than just the exception name. Its a lot more useful.

Reply:
Done